firmware_configurations/reprapfirmware/fulg/update.bat

```diff
@@ -0,0 +1,9 @@
+wget --ftp-user=admin -r -nH ftp://voronv2015.local/macros
+wget --ftp-user=admin -r -nH ftp://voronv2015.local/sys
+
```

Comment:
even though this is just a local IP, you may want to add a note here or replace the url with something generic.